subsys/spm/spm.c

```diff
@@ -29,11 +29,12 @@
 #define NON_SECURE_APP_ADDRESS DT_FLASH_AREA_IMAGE_0_NONSECURE_OFFSET_0
 #endif /* USE_PARTITION_MANAGER */
 
-#define FIRST_NONSECURE_ADDRESS (NON_SECURE_APP_ADDRESS)
-#define LAST_SECURE_REGION_INDEX \
-	((FIRST_NONSECURE_ADDRESS / FLASH_SECURE_ATTRIBUTION_REGION_SIZE) - 1)
+#define NON_SECURE_RAM_OFFSET 0x10000
```

Comment:
Sorry but I don't understand this definition. Shouldn't it be configurable?

Reply:
@hakonfam @sigvartmh The offset was never configurable, so this just rearranges things. I'd prefer to use a DTS definition instead, but there is none available with a sensible name (only `DT_MMIO_SRAM_20010000_BASE_ADDRESS`). I'll add a comment, something like "This reflects the configuration in DTS."

Reply:
So this will be changed when PM regions is in? As then this would be a known define.

Reply:
I assume so.

Reply:
Added comment now.